commons/tools/access-logs.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Access Logs
+shortdesc: Get and process your access logs.
+tags:
+- apps, metrics, accesslogs, warp10
+---
+
+# Access Logs analysis
+
+All your applications access logs are now pushed to Warp10. You are now able to process them directly on the console on the Metrics tab of your applications.
```

Comment:
C'est pas un blog post, donc pas de "are now", juste "now". Il faut que ce soit timeless.

Reply:
s/on/in/ aussi

Reply:
Les deux on to in?

Reply:
oui